sound/soc/sof/ipc4.c

```diff
@@ -342,6 +342,8 @@ static int ipc4_tx_msg_unlocked(struct snd_sof_ipc *ipc,
 	if (msg_bytes > ipc->max_payload_size || reply_bytes > ipc->max_payload_size)
 		return -EINVAL;
 
+	sof_ipc4_log_header(sdev->dev, "ipc tx      ", msg_data, true);
```

Comment:
@ujfalusi typo in the commit message, I think you mean "rather than after it sending" instead of "then after it has been sent"

Reply:
the ypo is still there @ujfalusi